pysus/online_data/Infodengue.py

```diff
@@ -0,0 +1,117 @@
+import json
+import string
+from difflib import get_close_matches
+from pathlib import Path
+from typing import Dict
+
+import pandas as pd
+import unidecode
+
+# from loguru import logger
+
+APP_DIR = Path(__file__).resolve(strict=True).parent.parent
+CID10 = {"dengue": "A90", "chikungunya": "A92.0", "zika": "A928"}
+
+with open(APP_DIR / "dataset/geocode_by_cities.json", "r") as f:
+    geocode_by_cities = json.load(f)
+
+
+def normalize(s):
+    for p in string.punctuation:
+        s = s.replace(p, '')
+
+    return unidecode.unidecode(s.lower().strip())
+
+
+def search_string(substr: str) -> Dict[str, int]:
+    """
+    Fetch geocode of the city name matching to the substring.
+
+    Parameters
+    ----------
+        substr: Part of city name
+    Returns
+    -------
+        dict: Dictionary with key and values
+            with city name and IBGE codes of all municipalities in Brazil
+    """
+    normalized_list = [normalize(f) for f in list(geocode_by_cities.keys())]
+    
+    cites_mathes = [
```

Comment:
I think there is a typo in the variable name...
Perhaps if should be named `matching_cities`?